awscli/customizations/s3/results.py

```diff
@@ -163,8 +164,11 @@ def __init__(self):
         self.expected_files_transferred = 0
 
         # Keys: result from _get_progress_cache_key()
-        # Values: number of bytes left for a particular transfer
-        self._progress_remaining_cache = {}
+        # Values: number of bytes that have been transferred for that key
+        self._progress_cache = defaultdict(int)
+        # Keys: result from _get_progress_cache_key()
```

Comment:
There is no longer a `_get_progress_cache_key()`.  

But more from a general coding practice standpoint, these comments aren't helpful.  I get that they key for the _process_cache dict is whatever is returned from `_get_process_cache_key()` because that's literally the name of the method.  I'd suggest either renaming these methods or just omitting these types of comments in general unless they're add insight not gleamed from reading the code.  They get out of date and don't help people reviewing the code.